scripts/valid_images.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@mellon-collie I think this file can be pushed along with the `push_image` change. Also, please verify the list once again since we removed a few images

Reply:
Done. I've merged this branch with the latest changes in master. 
Link to the successful GitHub action run: https://github.com/kanisterio/kanister/actions/runs/5413775340/jobs/9839845888